components/style/placementArrow.ts

```diff
@@ -85,15 +85,15 @@ export default function getArrowStyle<
           `&-placement-topRight > ${componentCls}-arrow`,
         ].join(',')]: {
           bottom: arrowDistance,
-          transform: 'translateY(100%) rotate(180deg)',
+          transform: 'translateY(95%) rotate(180deg)',
```

Comment:
dont do the magic code. It will fix this one but cause other brower issue. We don't handle the scale issue since browsers render behavior is not the same.